GitOut/Features/Git/Log/LogEntriesViewModel.cs

```diff
@@ -37,12 +37,23 @@ public LogEntriesViewModel(GitTreeEvent root, IGitRepository repository)
             Root = root;
             this.repository = repository;
 
-            allFiles = new SortedLazyAsyncCollection<IGitFileEntryViewModel>(() => ListAllFilesAsync(), IGitDirectoryEntryViewModel.CompareItems);
-            var logFiles = new SortedLazyAsyncCollection<IGitFileEntryViewModel>(() => GitFileEntryViewModelFactory.ListIdAsync(root.Event.Id, repository), IGitDirectoryEntryViewModel.CompareItems);
-            _ = logFiles.MaterializeAsync().AsTask();
+            allFiles = new SortedLazyAsyncCollection<IGitFileEntryViewModel, RelativeDirectoryPath>(_ => ListAllFilesAsync(), IGitDirectoryEntryViewModel.CompareItems);
+            var logFiles = new SortedLazyAsyncCollection<IGitFileEntryViewModel, RelativeDirectoryPath>(relativePath =>
+                GitFileEntryViewModelFactory.ListIdAsync(root.Event.Id, repository, relativePath),
```

Comment:
I think the lambda should be on one line here (move `relativePath` down in other words) to make it more clear that the ctor takes two parameters, one is the lambda and the other is CompareItems.
_Same below btw_

Reply:
Uhm, you only updated the other one ^^